meson.build

```diff
@@ -537,7 +537,9 @@ configure_file(
   configuration: conf
 )
 
-amdgpu = cc.has_header_symbol('drm/amdgpu_drm.h', 'AMDGPU_INFO_VBIOS_INFO', required: get_option('plugin_amdgpu'))
+libdrm_amdgpu = dependency('libdrm_amdgpu',
+                           version: '>= 2.4.113',
+                           required: get_option('plugin_amdgpu'))
```

Comment:
out of interest -- how much of amdgpu_drm.h do we actually need? if it's < 20 lines maybe copy and paste might be a good idea?

Reply:
We need probably about 50 lines from there, but it depends on drm.h which is a lot bigger.